app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -220,17 +226,29 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         themeManager.applyStatusBarTheme(this)
     }
 
-    private fun setupToolbarAndNavigation() {
+    /**
+     * Returns the [supportActionBar], inflating it if necessary.
+     * Name chosen to be more discoverable and avoid confusion with references to getToolbar
```

Comment:
nit: I'd include a comment about, "Everyone should call this instead of `supportActionBar`."

For the line I highlighted, maybe something like, "We chose this name because it was more discoverable than `getToolbar`... (though I'm not sure we need this line, it actually makes sense that it's an action bar and it's confusing to explain why we wanted to call it toolbar anyway)